### DISCUSSION TITLE
can teleport replace ngrok?

### DISCUSSION THREAD

**Discussion by ghost** [gravitational/teleport#16492] [Q&A]
We are using ngrok on a macbook so we can access it over ssh from a Jenkins server. We have already set up Teleport on the Jenkins server and the macbook and can use ssh via Teleport. Could we drop ngrok and use Teleport instead, and what would be the hostname of the macbook to substitute for the ngrok one?

**Comment by webvictim** [comment]
Yes - this sounds like you'd want to install the Teleport SSH service on the laptop and then configure Jenkins to use Machine ID to access it.

https://goteleport.com/docs/server-access/getting-started/

https://goteleport.com/docs/machine-id/getting-started/
